routes/schemas.js

```diff
@@ -47,9 +47,55 @@ module.exports = {
 		// we want it to be stored in MongoDB as a number
 		validUntil: Joi.number().required(),
 		creator: creator(cfg),
+		created: Joi.number(),
+		ipfs: Joi.string().regex(/^ipfs:/),
```

Comment:
these properties should definitely not be here; anything new should be in the spec